src/components/HighlightedContent.js

```diff
@@ -3,15 +3,15 @@ import React from 'react';
 // Function to highlight words within a sentence
 export const highlightWords = (sentence, query) => {
   const words = sentence.split(' ');
-  const highlightedIndices = []; // Array to store indices of highlighted words
+  //const highlightedIndices = []; // Array to store indices of highlighted words
```

Comment:
don't comment out code used for testing. to get around the unreferenced warnings, use _ or some warning suppression directive comment before the variable